### PR TITLE
main merge with docking backend

### PR DESCRIPTION
**Pull request by SanazKaz**

### CODE REVIEW

File: .DS_Store
Comment (SanazKaz) [inline .DS_Store:1]: should be in .gitignore

File: assay_ml_models/automl_logd_model.pkl
Comment (SanazKaz) [inline assay_ml_models/automl_logd_model.pkl:1]: shouldnt be removed

File: package-lock.json
Comment (SanazKaz) [inline package-lock.json:1]: deleete

File: static/.DS_Store
Comment (SanazKaz) [inline static/.DS_Store:1]: cannot find and should be ignored anyway

File: package-lock.json
Comment (SanazKaz) [reply package-lock.json:1]: deleted.

File: .DS_Store
Comment (SanazKaz) [reply .DS_Store:1]: done and removed.

File: static/.DS_Store
Comment (SanazKaz) [reply static/.DS_Store:1]: removed.

**Review by npqst — APPROVED** [review]
Looks good :)

**Pull request merged**